extensions-core/druid-basic-security/src/main/java/org/apache/druid/security/basic/CommonCacheNotifier.java

```diff
@@ -182,18 +213,43 @@ public void addUpdate(String updatedItemName, byte[] updatedItemData)
     return futures;
   }
 
-  private URL getListenerURL(DruidNode druidNode, String baseUrl, String itemName)
+  private List<ListenableFuture<StatusResponseHolder>> sendUpdate(byte[] serializedEntity)
```

Comment:
This sendUpdate method should be removed as a result of deleting the unused codepaths in this class

Reply:
Done, removed